### ISSUE TITLE
Gamemode indicator icon takes up space even when disabled

### ISSUE THREAD

**Issue by MithicSpirit** [Alexays/Waybar#2206]
The indicator icon for the gamemode module takes up space even if `use-icon: false` is set. This can be noticed by launching `GTK_DEBUG=interactive waybar`, finding the gamemode image in the object hierarchy, clicking on it, and noticing how there is a small blue box next to the text. Alternatively, putting a background on `#gamemode` in the CSS shows that there is more spacing to the left of the text than to the right, as that extra space is accounting for the icon (see image). Setting icon-size to 0 helps but does not fix the problem entirely (setting icon-spacing to 0 seems to have no effect).

![Gamemode module with uncentered text](https://github.com/Alexays/Waybar/assets/24192522/d0daa465-5207-4256-85b9-c4d014e551e6)

**Comment by ninetailedtori** [comment]
This one is still broken, did you find a way around it yet?

**Comment by MithicSpirit** [comment]
No, I just haven't bothered to set up a proper bar on my new gaming system yet. Still using stock swaybar (bleh).

**Comment by ninetailedtori** [comment]
Too real...I'm just gonna regress to the icon rather than glyph for now and see if I can patch it in a PR if I find the issue xD
